### PR TITLE
Removing rawtypes warnings by suppression - Iteration 1

### PR DESCRIPTION
**Pull request by neeravagarwalmicrosoft**
Task: Removing compile time warnings

Action taken: Suppressed many of the raw type warnings.

Testing: 
1. Common repository is compiling successfully with the changes
2. Tested these changes in OneAuth code by running the MSAL flow in Android test app.

Outcome:
Before this change: Common warnings: 645
After this change: Common warnings: 555

### CODE REVIEW

File: common/src/main/java/com/microsoft/identity/common/internal/migration/TokenCacheItemMigrationAdapter.java [hunk 1]
```diff
@@ -136,7 +136,7 @@
      * @throws ClientException
      * @throws IOException
      */
-    public static boolean tryFociTokenWithGivenClientId(@NonNull final BrokerOAuth2TokenCache brokerOAuth2TokenCache,
+    public static boolean tryFociTokenWithGivenClientId(@SuppressWarnings("rawtypes") @NonNull final BrokerOAuth2TokenCache brokerOAuth2TokenCache,
                                                         @NonNull final String clientId,
```
Comment (saugup-ms) [inline common/src/main/java/com/microsoft/identity/common/internal/migration/TokenCacheItemMigrationAdapter.java:140]: Use  @SuppressWarnings(rawtype_warning) as you did everywhere.

Comment (neeravagarwalmicrosoft) [reply common/src/main/java/com/microsoft/identity/common/internal/migration/TokenCacheItemMigrationAdapter.java:140]: Sure, I'll do.

File: common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java [hunk 2]
```diff
@@ -93,6 +93,8 @@
 
 public abstract class BaseController {
 
+    // Declaring rawtypes warning name to avoid using the literal many times in this class
+    private static final String rawtype_warning = "rawtypes";
```
Comment (shoatman) [inline common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java:97]: I like the use of the constant; however can we use this everywhere?

**Review by shoatman — APPROVED** [review]
Just left the comment about using the constant everywhere.  Feel free to add a specific class for all suppression constants to common.

File: common/src/main/java/com/microsoft/identity/common/internal/migration/TokenCacheItemMigrationAdapter.java (hunk 1 above)
Comment (neeravagarwalmicrosoft) [reply common/src/main/java/com/microsoft/identity/common/internal/migration/TokenCacheItemMigrationAdapter.java:141]: I have taken care of this. Thanks

File: common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java (hunk 2 above)
Comment (neeravagarwalmicrosoft) [reply common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java]: Sure @shoatman , I have now used the constant everywhere. Thanks.

**Comment by neeravagarwalmicrosoft** [comment]
I have taken care of the comments!

**Review by jaiswalh — APPROVED** [review]

**Pull request merged**